### PR TITLE
added cryptonote.social api

### PR DESCRIPTION
**Pull request by andrewnk**

### CODE REVIEW

File: v2/turtlecoin-pools.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
         {
             "name" : "CryptoNote.social",
             "url" : "https://cryptonote.social/trtl/",
+            "api" : "https://cryptonote.social/pool/statistics/trtl",
             "type" : "other",
```
Comment (zpalmtree) [inline v2/turtlecoin-pools.json]: Maybe we should change the type to a cryptonote.social or something, so it doesn't conflict with other, `other` pools?

Comment (brandonlehmann) [reply v2/turtlecoin-pools.json]: This is the only `other` currently listed... so what's it gonna conflict with?

Comment (zpalmtree) [reply v2/turtlecoin-pools.json]: I swear there used to be another one.

Comment (brandonlehmann) [reply v2/turtlecoin-pools.json]: 🤗

**Pull request merged**